Samples/Target group condition Login status/Resources/LoginStatusSetup.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using Litium.Web.Administration.Filtering;
+using Litium.Studio.Extenssions;
+using Litium.Runtime.AutoMapper;
+using AutoMapper;
+using Litium.Owin.Lifecycle;
+using Litium.Foundation;
+using Litium.Web.Customers.TargetGroups;
+using Litium.Events;
+
+namespace Litium.Accelerator.TargetGroupconditions.LoginStatus
+{
+    /// <summary>
+    /// Setup class for Login status target group condition.
+    /// Setup is triggered on application start and is used to register Automapper-mapping required by the condition
+    /// and to register target group event to trigger every time a user
+    /// </summary>
+    public class LoginStatusSetup : IStartupTask, IAutoMapperConfiguration
```

Comment:
Use the `[Autostart]` attribute instead of `IStartupTask` and move the logic from the start-method into the ctor.

Reply:
What is the benefit of Autostart over IStartupTask?

Reply:
Old and new ways to do the same thing on. `IStartupTask`= old way.